src/js/components/Select/index.d.ts

```diff
@@ -7,14 +7,14 @@ import {
   MarginType,
   PlaceHolderType,
 } from '../../utils';
-
+type disabledKeyType = string | undefined | object | boolean | Function;
```

Comment:
Looking at the docs it doesn't look like we support object or boolean for this prop

Reply:
In the File \src\js\components\Select\utils.js, we have a function applyKey in which the second argument we are passing is the disabledkey and there we are checking if the typeof Key === 'object'. That is why I have added a type of object.

For the Boolean Type, I made the same mistake of thinking that as we are using it in if condition it can be boolean

I have made the relevant changes. Thanks